packages/fluentui/docs/src/examples/components/Card/Slots/CardExamplePreview.tsx

```diff
@@ -2,7 +2,7 @@ import { Card, Image } from '@fluentui/react-northstar';
 import * as React from 'react';
 
 const CardExamplePreview = () => (
-  <Card compact>
+  <Card compact ariaRoleDescription="image card">
```

Comment:
```suggestion
  <Card compact aria-roledescription="image card">
```

Please use native prop instead of custom one